docs/concepts/index.md

```diff
@@ -9,6 +9,9 @@ sidebar_position: 3
 
 Core concepts, what is STRM Privacy, what problems are we solving?
 
+## [Data Subjects](data-subjects.md)
```

Comment:
I've removed this section in the new docs revamp, but you can leave it in for now.